src/control/lib/telemetry/telemetry.go

```diff
@@ -124,6 +125,17 @@ func findNode(hdl *handle, name string) (*C.struct_d_tm_node_t, error) {
 	return node, nil
 }
 
+func getPathAndName(fullName string) (string, string) {
```

Comment:
passing name into getPathAndName() seems a bit counterintuitive, not sure it's much better but getPathAndShortName() or splitFullName() might be more readable?

Reply:
splitFullName sounds good. I can address in a follow-on, so we can get the basic functionality landed ASAP.

Reply:
Fixed in #6200